lib/salus/report.rb

```diff
@@ -187,9 +136,44 @@ def export_report
 
     private
 
-    # Wrap lines to 100 chars by default
-    def wrap(text, line_width = 100)
-      text.gsub(/(.{1,#{line_width}})/, "\\1\n")
+    def deep_stringify_keys(datum)
+      case datum
+      when Hash then datum.map { |key, value| [key.to_s, deep_stringify_keys(value)] }.to_h
+      when Array then datum.map { |value| deep_stringify_keys(value) }
+      else datum
```

Comment:
do you need a `.to_s` on this?

Reply:
nope - integers should stay integers, etc. could add a `when Symbol` case for paranoia though